### PR TITLE
msggen: cluster add methods 1

### PR DESCRIPTION
**Pull request by daywalker90**
adds:

- parsefeerate
- plugin
- renepay
- renepaystatus
- sendinvoice

Introduces new primitive `PluginSubcommand`. 

`sendinvoice`'s `label` changed to only `string` in the schema.

### CODE REVIEW

File: .msggen.json [hunk 1]
```diff
@@ -4196,70 +4185,6 @@
...
-            "deprecated": false
-        },
-        "Fundchannel_Start.scriptpubkey": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
-        "Fundchannel_Start.warning_usage": {
-            "added": "pre-v0.10.1",
-            "deprecated": false
-        },
```
Comment (cdecker) [inline .msggen.json:4262]: Oops, this looks like it'll need a rebase. Let me try to rebase it myself, and see if we end up with this as well.

Comment (daywalker90) [reply .msggen.json:4262]: sorry about this one, we merged a bad .msggen.json into master where i generated Fundchannel and FundChannel. So every pr now has these deletions in them.

Comment (daywalker90) [reply .msggen.json:4262]: but the deletions are merged aswell already so...

Comment (cdecker) [reply .msggen.json:4262]: Ok, got it. As long as the IDs don't change in-between releases, that's fine too. :+1:

**Pull request merged**